pkgs/development/libraries/libphonenumber/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl, cmake, gmock, boost, pkgconfig, protobuf, icu }:
+
+let
+  version = "8.9.9";
+in
+stdenv.mkDerivation {
+  name = "phonenumber-${version}";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://github.com/googlei18n/libphonenumber/archive/v${version}.tar.gz";
+    sha256 = "1m2z5hk2yg99sqi70wf26biprhppklwxwyk0g717c1x6yybch13g";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    gmock
+    boost
+    pkgconfig
+    protobuf
+    icu
+  ];
+
+  meta = with stdenv.lib; {
+    description = "Google's i18n library for parsing and using phone numbers";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ illegalprime ];
+  };
```

Comment:
Nitpick: Can you move this meta section to the very bottom? It's somewhat of a convention